extensions/wikia/WikiaMaps/controllers/WikiaMapsParserTagController.class.php

```diff
@@ -202,12 +200,12 @@ public function validateParseTagParams( Array $params, &$errorMessage ) {
 			return $isValid;
 		}
 
-		foreach( $params as $param => $value ) {
-			$isValid = $this->isTagParamValid( $param, $value, $errorMessage );
+		$errorMessages = $this->validateAttributes( $params );
 
-			if( !$isValid ) {
-				return $isValid;
-			}
+		if( !empty( $errorMessages ) ) {
+			$errorMessage = $errorMessages[0];
+		} else {
+			$isValid = true;
```

Comment:
It seems that variable `$isValid` is not needed any more. What do you think about returning `true`/`false` and removing here this `else` statement?